ctest-next/templates/test.rs

```diff
@@ -13,6 +13,8 @@ mod generated_tests {
     use std::fmt::{Debug, LowerHex};
     use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
     use std::{mem, ptr, slice};
+    #[expect(unused_imports)]
+    use std::mem::{MaybeUninit, offset_of};
```

Comment:
Shouldn't this be `allow` rather than `expect`? Because when there are field tests, it's not actually unused.